service/test/_client_common_testrun.js

```diff
@@ -88,12 +88,53 @@ function transportSpecificTests(opts) {
       });
     });
 
+    describe('#sendToModule', function () {
+      function createTestMessage() {
+        var msg = new Message('msg');
+        msg.expiryTimeUtc = Date.now() + 5000; // Expire 5s from now, to reduce the chance of us hitting the 50-message limit on the IoT Hub
+        return msg;
+      }
+
+      /*Tests_SRS_NODE_IOTHUB_CLIENT_18_009: [When the `sendToModule` method completes, the callback function (indicated by the done - argument) shall be invoked with the following arguments:
+      - `err` - standard JavaScript Error object (or subclass)
+      - `result` - an implementation-specific response object returned by the underlying protocol, useful for logging and troubleshooting]*/
+      /*Tests_SRS_NODE_IOTHUB_CLIENT_18_010: [The argument `err` passed to the callback `done` shall be `null` if the protocol operation was successful.]*/
+      it('sends the message', function (done) {
+        testSubject.send(deviceId, createTestMessage(), function (err, state) {
+            if (!err) {
+            assert.equal(state.constructor.name, "MessageEnqueued");
+            }
+            done(err);
+        });
+      });
+
+      /*Tests_SRS_NODE_IOTHUB_CLIENT_18_008: [The `sendToModule` method shall convert the message object to type azure-iot-common.Message if necessary.]*/
+      it('accepts any message that is convertible to type Message', function (done) {
+        var message = 'msg';
+        testSubject.send(deviceId, message, function (err, state) {
+            if (!err) {
+            assert.equal(state.constructor.name, "MessageEnqueued");
+            }
+            done(err);
+        });
+      });
+
+    /*Tests_SRS_NODE_IOTHUB_CLIENT_18_012: [If the `deviceId` has not been registered with the IoT Hub, `sendToModule` shall call the `done` callback with a `DeviceNotFoundError`.]*/
+    it('returns DeviceNotFoundError when sending to an unregistered deviceId', function (done) {
+        var unregisteredDeviceId = 'no-device' + Math.random();
```

Comment:
>'no-device' [](start = 35, length = 11)

if keeping it let's add a comment there explaining why it'll fail during unit-tests